content/python/concepts/loops/loops.md

```diff
@@ -118,6 +118,18 @@ for i in big_number_list:
   print(i)
 ```
 
+## Pass Keyword
+
+The `pass` keyword is used as a placeholder statement to allow empty functions or classes to be included in an executable code block without throwing an error. This is common when structuring future implementations.
+
+```py
+def my_function():
+  pass  # Placeholder for future implementation
+
+class MyClass:
+  pass  # Placeholder for an empty class
+```
```

Comment:
We should explain `pass` here with an example that uses loops. It can be something like this - 

```py
# Example: Filtering even numbers but doing nothing with them
numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

for num in numbers:
    if num % 2 == 0:
        # Placeholder for future logic to handle even numbers
        pass
    else:
        print(f"Odd number: {num}")
```